src/components/sbb-checkbox/sbb-checkbox.tsx

```diff
@@ -225,67 +236,80 @@ export class SbbCheckbox implements ComponentInterface {
-                  indeterminate={this.indeterminate}
-                  disabled={this.disabled || this._disabledFromGroup}
-                ></sbb-visual-checkbox>
-              </span>
-              <span class="sbb-checkbox__label">
-                <slot />
-                {(this.iconName || (this._namedSlots['icon'] && !this._isSelectionPanelInput)) && (
-                  <span class="sbb-checkbox__label--icon">
-                    <slot name="icon">{this.iconName && <sbb-icon name={this.iconName} />}</slot>
-                  </span>
-                )}
-                {!!this._selectionPanelElement && this._namedSlots['suffix'] && (
-                  <slot name="suffix" />
-                )}
-              </span>
+              }
+            })}
+            type="checkbox"
+            aria-hidden="true"
+            tabindex=${-1}
+            ?disabled=${this.disabled || this._disabledFromGroup}
+            ?required=${this.required || this._requiredFromGroup}
+            ?checked=${this.checked}
+            .value=${this.value || nothing}
+            @input=${() => this.handleInputEvent()}
+            @change=${(event) => this.handleChangeEvent(event)}
+            @focus=${() => this.focus()}
+          />
+          <span class="sbb-checkbox__inner">
+            <span class="sbb-checkbox__aligner">
+              <sbb-visual-checkbox
+                ?checked=${this.checked}
+                ?indeterminate=${this.indeterminate}
+                ?disabled=${this.disabled || this._disabledFromGroup}
+              ></sbb-visual-checkbox>
             </span>
-            {!!this._selectionPanelElement && this._namedSlots['subtext'] && (
-              <slot name="subtext" />
-            )}
-            {this._isSelectionPanelInput && this._selectionPanelExpandedLabel && (
-              /* For screen readers only */
-              <span class="sbb-checkbox__expanded-label">{this._selectionPanelExpandedLabel}</span>
-            )}
-          </label>
-        </span>
-      </Host>
-    );
+            <span class="sbb-checkbox__label">
+              <slot></slot>
+              ${this.iconName || (this._namedSlots['icon'] && !this._isSelectionPanelInput)
+                ? html`<span class="sbb-checkbox__label--icon">
+                    <slot name="icon">
+                      ${this.iconName
+                        ? html`<sbb-icon name="${this.iconName}"></sbb-icon>`
+                        : nothing}
+                    </slot>
+                  </span>`
+                : nothing}
+              ${!!this._selectionPanelElement && this._namedSlots['suffix']
+                ? html`<slot name="suffix"></slot>`
+                : nothing}
+            </span>
+          </span>
+          ${!!this._selectionPanelElement && this._namedSlots['subtext']
+            ? html`<slot name="subtext"></slot>`
+            : nothing}
+          ${this._selectionPanelElement && this._selectionPanelExpandedLabel
```

Comment:
It seems that the commit where I fixed a bug was not correctly transferred to checkbox. e.g. here should be 
```suggestion
          ${this._isSelectionPanelInput && this._selectionPanelExpandedLabel
```